addon/src/test-support/get-page-title.ts

```diff
@@ -1,17 +1,17 @@
 // Testem appends progress to the title...
 // and there's no way to stop this at the moment
 
-export function getPageTitle(doc: Document) {
+export function getPageTitle(doc?: Document): string {
```

Comment:
`doc` should not be required because of `doc || window.document`.

Reply:
Current return type is `string | false | undefined` because of the current implementation. This doesn't seem ideal.